playbooks/install_stack.yaml

```
@@ -456,7 +481,7 @@
   - name: Reboot if SR-IOV is enabled (to apply kernel changes)
```

It can reboot the node even if SR-IOV is not enabled. I suggest you rename the task:
```suggestion
  - name: Reboot to apply kernel changes
```